# Description: don't tweak wrapping after dots, it's not required and 
# creates some issues (i.e with Ubuntu version number)
# Upstream: https://bugzilla.gnome.org/show_bug.cgi?id=674924
#
Index: nautilus-3.8.2/libnautilus-private/nautilus-canvas-item.c
===================================================================
--- nautilus-3.8.2.orig/libnautilus-private/nautilus-canvas-item.c	2014-01-15 14:31:48.369605599 +1300
+++ nautilus-3.8.2/libnautilus-private/nautilus-canvas-item.c	2014-01-15 14:31:48.365605599 +1300
@@ -1405,11 +1405,6 @@
 
 #define ZERO_WIDTH_SPACE "\xE2\x80\x8B"
 
-#define ZERO_OR_THREE_DIGITS(p)			\
-	(!g_ascii_isdigit (*(p)) ||		\
-	 (g_ascii_isdigit (*(p+1)) &&		\
-	  g_ascii_isdigit (*(p+2))))
-
 
 static PangoLayout *
 create_label_layout (NautilusCanvasItem *item,
@@ -1438,11 +1433,9 @@
 		for (p = text; *p != '\0'; p++) {
 			str = g_string_append_c (str, *p);
 
-			if (*p == '_' || *p == '-' || (*p == '.' && ZERO_OR_THREE_DIGITS (p+1))) {
+			if (*p == '_' || *p == '-' || (*p == '.' && !g_ascii_isdigit(*(p+1)))) {
 				/* Ensure that we allow to break after '_' or '.' characters,
-				 * if they are not likely to be part of a version information, to
-				 * not break wrapping of foobar-0.0.1.
-				 * Wrap before IPs and long numbers, though. */
+				 * if they are not followed by a number */
 				str = g_string_append (str, ZERO_WIDTH_SPACE);
 			}
 		}
